IFC4x3/Properties/n/NValue_0ODh2KpO5Awv4vDj6gcncD/Documentation.md

```diff
@@ -0,0 +1 @@
+N=N2+N3 is the result of the test.
```

Comment:
But what is `NValue`? I would expect a description of `NValue` in the description of the NValue property. Please provide a better description.